### ISSUE TITLE
screen flip with rtl addon half way 

### ISSUE THREAD

**Issue by sopdevv** [notion-enhancer/desktop#931]
**I downloaded the files by my self and added it to chrom,I added the rtl addon, I think notion updated their ui this week, so the rtl addon is diabled or stop working when i scroll down in the main page, and when i go up it starts working again**

**What should be happening?: rtl complete working even after scroll**

**windows 10/chrome/Version 130.0.6723.59 (Official Build) (64-bit)**

**notion-enhancer but i downloded by myself not the released version**

**notion-enhancer configuration: rtl extension, Word Counter extension**

keep in mind it was working good before notion latest ui changes

**Comment by yaya1426** [comment]
For me the notion-enhancer causes notion to crash chrome. Not sure if anyone else is suffering same issue too.

**Comment by sopdevv** [comment]
yeah I already reported this bug and it's already fixed, but the creator want to realese it with bunch of other things that he is still working on, see https://github.com/notion-enhancer/notion-enhancer/issues/929
